coala_quickstart/generation/Settings.py

```diff
@@ -0,0 +1,84 @@
+import os
+from collections import OrderedDict
+
+from pyprint.NullPrinter import NullPrinter
+from coalib.output.printers.LogPrinter import LogPrinter
+from coala_utils.Extensions import exts
+from coala_quickstart.generation.Utilities import (
+    split_by_language, get_extensions, get_gitignore_glob)
+from coalib.settings.Section import Section
+from coalib.output.ConfWriter import ConfWriter
+from coalib.collecting.Collectors import collect_files
+
+
+def generate_section(section_name, extensions_used, bears):
+    """
+    Generates a section for a particular language (or default).
+
+    :param section_name:
+        Name of the section.
+    :param extensions_used:
+        A list of extensions associated with this section.
+    :param bears:
+        A list of bear classes.
+    :return:
+        A ``Section`` object containing the section.
+    """
+    section = Section(section_name, None)
+
+    section["bears"] = ", ".join(bear.name for bear in bears)
+    section["files"] = ", ".join("**" + ext for ext in extensions_used)
```

Comment:
Nope, now we're taking all files that are tracked by git of a particular language for each section